__tests__/app.test.js

```diff
@@ -334,3 +334,33 @@ describe("GET:/api/users", () => {
       });
   });
 });
+describe("GET:/api/users/:username", () => {
+  it("200: Returns the data of a user using a given username", () => {
+    return request(app)
+      .get("/api/users/lurker")
+      .expect(200)
+      .then(({ body: { usernameData } }) => {
+        expect(usernameData).toMatchObject({
+          username: expect.any(String),
+          avatar_url: expect.any(String),
+          name: expect.any(String),
+        });
+      });
+  });
+  it("400: Should return an error if a wrong input is used in the url", () => {
+    return request(app)
+      .get("/api/users/100")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("incorrect input");
+      });
```

Comment:
If this is a `VARCHAR` data type then a string of _'100'_ isn't technically invalid because it could be any string characters